js-client-sdk.api.md

```diff
@@ -212,7 +212,9 @@ export interface IPrecomputedClientConfigSync {
     throwOnFailedInitialization?: boolean;
 }
 
-// @public (undocumented)
+// Warning: (ae-internal-missing-underscore) The name "NO_OP_EVENT_DISPATCHER" should be prefixed with an underscore because the declaration is marked as @internal

```

Comment:
Lint warning via doc generation. fun. really we should be moving all the declarations out of index.ts and have only the exports.